src/components/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,38 @@
+@import 'variables.scss';
+
+.c-side-bar {
```

Comment:
My suggestion is to name this `.c-sidebar` no need for `-` but it is just suggestion